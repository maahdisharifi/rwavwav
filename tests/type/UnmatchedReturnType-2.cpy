float getNothing():
    return
end

int main():
    float f = getNothing()
    return 0
end