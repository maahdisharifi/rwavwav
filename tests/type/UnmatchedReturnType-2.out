Line:2 -> return type mismatch